README.md

```diff
@@ -21,6 +21,24 @@ If you wish to see visualisations of parse trees, you may optionally install:
 * PyDot https://code.google.com/p/pydot/
 * Pygame https://www.pygame.org/
 
+#### On Ubuntu (including Windows Subsysten for Linux) ####
+
+The above dependencies can be installed as follows on command line:
+
+```
+# Get Python 2 *and* 3 if you haven't already

```

Comment:
Python3 probably suffices here as that is what most people need unless they want to export Python programs.